tests/data/context/capability.yaml

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: 2022 Copyright DB Netz AG and the capellambse-context-diagrams contributors
+# SPDX-License-Identifier: Apache-2.0
+
+graph: targets * exchanges -> Capability -> exchanges * targets
+
+subject: Capability
```

Comment:
It looks like the names of these YAML files are supposed to be what it says for the `subject:`. In that case, I'm for keeping them 1:1 the same instead of lower-casing the file names. That just makes things slightly easier to read and associate.